Fb1149

spireken opened the pull request:
I put the builds in the misc_builds folder on Dropbox.

kranar commented:
Test that the minimize fading animation works.

On Fri, Jul 20, 2018, 12:52 PM spireken <notifications@github.com> wrote:

> I put the builds in the misc_builds folder on Dropbox.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/eidolonsystems/nexus/pull/224
> Commit Summary
>
>    - Added window flag, removed redundant flag
>    - Removed Qt::Window flags
>
> File Changes
>
>    - *M*
>    Applications/spire/source/book_view/book_view_properties_dialog.cpp
>    <https://github.com/eidolonsystems/nexus/pull/224/files#diff-0> (2)
>    - *M* Applications/spire/source/login/login_window.cpp
>    <https://github.com/eidolonsystems/nexus/pull/224/files#diff-1> (2)
>    - *M*
>    Applications/spire/source/security_input/security_input_dialog.cpp
>    <https://github.com/eidolonsystems/nexus/pull/224/files#diff-2> (2)
>    - *M*
>    Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp
>    <https://github.com/eidolonsystems/nexus/pull/224/files#diff-3> (2)
>    - *M* Applications/spire/source/toolbar/toolbar_window.cpp
>    <https://github.com/eidolonsystems/nexus/pull/224/files#diff-4> (2)
>    - *M* Applications/spire/source/ui/drop_shadow.cpp
>    <https://github.com/eidolonsystems/nexus/pull/224/files#diff-5> (2)
>
> Patch Links:
>
>    - https://github.com/eidolonsystems/nexus/pull/224.patch
>    - https://github.com/eidolonsystems/nexus/pull/224.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/224>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGXchxbUyRMovPmMuaYVzDhgJs-QXks5uIgrfgaJpZM4VYa_8>
> .
>

spireken commented:
It shrinks and slides towards the task bar.

kranar commented:
Okay can you upload a demo and I'll go over it when I get back to the
office.

On Fri, Jul 20, 2018, 12:55 PM spireken <notifications@github.com> wrote:

> It shrinks and slides towards the task bar.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/224#issuecomment-406661447>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGX9CjddMOqZ2EGx51nAun5m0ymi0ks5uIguagaJpZM4VYa_8>
> .
>

kranar commented:
What specific application do I run?

On Fri, Jul 20, 2018 at 1:24 PM, spireken <notifications@github.com> wrote:

> Yep, I put all of the affected builds in the misc_builds folder.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/224#issuecomment-406669632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGU0aEZ5-Qph1TSv8Mm2YuZfAIkisks5uIhJlgaJpZM4VYa_8>
> .
>



-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

spireken commented:
The toolbar_ui_tester is the most relevant one. I removed the Qt::Window flags from other windows so I added those builds to show that id didn't affect anything.
